src/auth/__tests__/authService.test.js

```diff
@@ -25,9 +24,10 @@ describe('authService', () => {
 
   describe('getToken', () => {
     it('should get API_TOKENS from localStorage', () => {
+      const getSpy = jest.spyOn(Storage.prototype, 'getItem');
       authService.getToken();
 
-      expect(localStorage.getItem).toHaveBeenNthCalledWith(3, API_TOKEN);
+      expect(getSpy).toHaveBeenNthCalledWith(1, API_TOKEN);
```

Comment:
It does not makes sense why getItem would be called 3 times in this test. Maybe previous spy did not reset properly between tests.